### DISCUSSION TITLE
Suggestion: option for scale of the world/cities & etc. vs. trains/stations & etc.

### DISCUSSION THREAD

**Discussion by mariodk** [voxeltycoon/issues#347] [General]
1 main pet peeve i have in basically all these games: transport fever, OpenTTD, Chris Sawyer's Locomotion 
i kinda wish there was Options for

is the scaling of the world/cities vs. the trains+stations:
-transport fever: a main Station/bigger Train junction: filling ½ of the same space as a "City" also the maps is kinda too small for this game a big high speed train junction Will fill up very much % of the limited space map you have : 1 of the reasons i dropped this game and going back to OpenTTD/Chris Sawyer's Locomotion and hopefully this game in the future
-OpenTTD, Chris Sawyer's Locomotion /Voxel Tycoon: a "City" : under 50 buildings 
-both: a like 120m train station is half the lengh of the "City"

so the scaling in these games is kinda off
i dont mind the scaling is off
But do i have to believe that this town barely 2x bigger than a 120meter train station is a "city"?
Cities is Kilometers long from 1 end to the other end
even mid towns size can be that
and also the range between towns/cities too 
Cities is not "meters" long not even a town is that often they are more like villages then towns/Cities in these games

a "City" in those games and the size of it / amount of buildings
would be a small/mid size town in city skyline
so i would really like some kind of mode/setting/mod(possible with a slider for make it scaling of the world vs. trains/stations & etc.) to your liking
like a slider that go from 
0%: Tycoon games world/town scaling, 
50%: mid between Tycoon games scaling to city skyline,
100% mostly real world/city skyline like scaling

so hope for a options/setting like this dont have to replace the current scaling since people like it how it is but others want it maybe 50% or 100%

like if scaling setting is on 25%
the speed of each vehicles/conveyor belt will be 25% faster,
the capacity of each factory/Town´s people/vehicles goods limit will be 25%
the regions/towns/resouces areas will be 25% bigger and range between towns/resouces areas too
But the size of stations/vehicles/factory stuff is still same
so we can finnaly play a Transport tycoon like game with the scaling of city skyline or a mid between via a option like this 
heck might aswell make each of these things its own custom setting to make it very customizable

like a "city" in the game can be the size of a small City in a save game of city skyline 
and a 200m 2track train station will fill as much space as one equal to that in city skyline would in the world

i saw on the Roadmap "Subways" for later on
i personality dont think a Subway or Trams is much needed in a "City" not much bigger than a train station 
but they will be alot more needed if the world scaling vs. trains/stations & etc. gets alot bigger via a setting
when i played OpenTTD/Loco i honestly never needed Trams/Subways/not even that much busses running in the city/town at mid/end game
i do abit in transport fever but its not much i need it here either 
becuase of the scaling of world/towns vs. stations/Vehicles 

also idk if freight train / passenger trains can get a shared station(official or mods) in the future
but if not that would require: 2 train stations + possible track layout/junctions for it
and that will 100% sure be bigger than a "City"

ofc for this to work the game needs to be more "done"
might need oneway roads for a bigger city´s roads like them with +4 lanes(2 for both ways)
so those cities can look like cities & function like a city for road layout and not just big Towns with a 2lane(both ways) roads only
and the regions you start on and unlock also needs to be bigger to make the scale of Cities/Towns vs. stations/trains

maybe with this get done and scaling of the world/towns vs. stations/trains option to be like this
via mods atleast we can get "traffic" from normal cars/people in the city/towns and between city/towns if linked via roads
and ofc more public transport for people less cars in the roads

but again this idea is not for Right now but more down the line when the game is alot more done 
dont have to upscale the models/asset of the game 
just the world/towns/regions and maybe make the train/car/ship/airplane speeds match that scaling too and also match range of stations, how much goods on each Vehicles & etc. that matches the scaling of the world too

i do feel like this idea would be best to be in base game and not as a mod
so future mods can support this "mode/Setting of new game"

**Comment by dabartos** [comment]
I agree, from the first moment I started playing the game it's becoming an issue whenever I get into the "build mode." The scaling issue creeps in every time and it's bothersome to say the least. 
Trains are way too big in relation to cities and I feel like there's a lot of potential lost on the scaling being this much off, it basically cuts off the city's effect on the game as a whole.
